env/main/files/galaxy/config/user_preferences.yml

```
@@ -37,3 +37,17 @@ preferences:
               type: text
               required: False
               value: https://auth.quantum-computing.ibm.com/api
+    chatgpt:
+        description: ChatGPT
+        inputs:
+            - name: api_key
+              label: OpenAI API Key
+              type: password
+              required: False
+      groq:
```

```suggestion
    groq:
```
Looks like the indentation is off here